Sample JSON Rules/change_light_color_and_between_time_of_day.json

```diff
@@ -0,0 +1,79 @@
+{
+    "name": "If the light is turned on between sunrise and 1 hour after sunrise change the light color",
+    "actions": [
+        {
+            "if":{
+                "and":[
+                    {
+                        "between":{
+                            "value": {
+                                "time": {
+                                    "reference": "Now"
+                                }
+                            },
+                            "start": {
+                                "time": {
+                                    "reference": "Sunrise"
+                                }
+                            },
+                            "end": {
+                                "time": {
+                                    "reference": "Sunrise",
+                                    "offset": {
+                                        "value": {"integer":60},
```

Comment:
Since the Rule name **references 1 Hour** specifically, I'd suggest maintaining consistency updating the offset, _i.e._:

```
{
  "offset": {
    "value": { "integer": 1 },
    "unit": "Hour"
  }
}
```